sentient-core/test_research_agent_integration.py

```
@@ -0,0 +1,215 @@
+# Ensure we're in the right working directory
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+# Import after setting working directory
+from core.agents.research_agent import ResearchAgent
+from core.services.memory_service import MemoryService
+from app import app
+
+# Create a FastAPI test client
+client = TestClient(app)
+
+async def test_research_agent_direct():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)